raiden/tests/utils/tester_client.py

```diff
@@ -602,13 +602,15 @@ def deposit(self, amount):
         self.proxy.deposit(amount)
         self.tester_state.mine(number_of_blocks=1)
 
+    # opened() and closed() method return different values for geth and tester chain
```

Comment:
Hmm, that is strange. Without the fix I've got it failing in the new TC scenario. Maybe version of some of the libraries differs? I'll try it in a clean virtualenv and will report back.